src/sqlacodegen/cli.py

```diff
@@ -58,7 +86,20 @@ def main() -> None:
         action="store_true",
         help="ignore views (always true for sqlmodels generator)",
     )
+    parser.add_argument(
```

Comment:
So, we wouldn't want to have every dialect specific as a parser argument. We would wind up with a lot of unrelated options and it'll be hard to keep track of. The best option would be to add a multiple option parameter, so we can just pass custom arguments to `create_engine` as the need arises. This is what I originally had in mind.
Something like `--engine-arg key=value`, or for your use case --engine-arg `thick_mode=...`